docs/solutions/hypervisors/kvm/README.md

```diff
@@ -0,0 +1,20 @@
+KVM Virtualization
+===============================
+This folder contains sample terraform templates to allow you to run a KVM hypervisor on Oracle Cloud Infrastructure, based on the [Installing and Configuring KVM on Bare Metal Instances with Multi-VNIC] (https://docs.us-phoenix-1.oraclecloud.com/Content/Resources/Assets/installing_kvm_multi_vnics.pdf) white-paper and [A Simple Guide to Nested KVM Virtualization on Oracle Cloud Infrastructure](https://blogs.oracle.com/cloud-infrastructure/nested-kvm-virtualization-on-oracle-iaas) blog post.
```

Comment:
The link for [Installing and Configuring KVM on Bare Metal Instances with Multi-VNIC] isn't showing up correctly - probably because of the space between ] and (.